### DISCUSSION TITLE
Cannot comment on this ticket

### DISCUSSION THREAD

**Discussion by zillnet-l2** [community/community#114548] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hey there,

I submitted a ticket to GitHub support because my profile got flagged. Someone responded, asking me to change my email address.

I followed their instructions, but now I can't reply to the ticket, as you can see:
![alert](https://github.com/community/community/assets/164290572/9c36e524-4822-4abd-a6f3-733a20493110)

Here's the problem: If I don't change my email address, I can add comments, but if I do change my email, I can't add comments. This is ridiculous.

And another thing, waiting 7 days for a response to the Reinstatement Request is too long, isn't it? Does that mean waiting 14 days for two replies? Goodness, could someone assist me in getting this done more quickly?

**Comment by Favio2424** [comment]
same issue

**Comment by ghost** [comment]
Same issue how did you solve?
